### ISSUE TITLE
False positive for commented-out code

### ISSUE THREAD

**Issue by paulschreiber** [squizlabs/PHP_CodeSniffer#2342]
This line was flagged as "37% valid code":
```
		// return empty list if nothing found
```

**Comment by jrfnl** [comment]
4 of the 6 words used are words which would be recognized as PHP keywords, so I doubt there is any that can be done in the sniff to "fix" this.

Suggestions:
* Rephrasing to something more grammatically like "proper" English such as `// Returns an empty list when nothing is found.` will probably help.
* Alternatively, you could try to raise the threshold used by the sniff. See: https://github.com/squizlabs/PHP_CodeSniffer/wiki/Customisable-Sniff-Properties#squizphpcommentedoutcode
